test/release/rollup-ts.sh

```diff
@@ -6,18 +6,22 @@ UGLIFY_OPTIONS=$@
 minify_in_situ() {
     ARGS="$UGLIFY_OPTIONS --validate --in-situ"
     DIRS="$1"
+    echo '> esbuild' $DIRS
+    for i in `find $DIRS -type f -name '*.ts' | grep -v '\.d\.ts'`
+    do
+        echo "$i"
+        CODE=`cat "$i"`
+        node_modules/.bin/esbuild --loader=ts --target=es2019 > "$i" <<EOF
+$CODE
+EOF
+        ARGS="$ARGS $i"
+    done
```

Comment:
Separated out `esbuild` as its own pass, and lump it all together below for `--in-situ` to make SpiderMonkey round-trip testing work.